rendering/glb.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: glb
```

Comment:
Isn't it better to call the format `gltf` or `gltf_binary`? It seems to be much more common name than `glb`. From what I've seen, `.glb` is only used as a file extension.

The `/meta/id` should reflect the format name rather than file extension.

Reply:
gltf_binary seems better, will rename to that